tools/assets_database/utils.py

```diff
@@ -30,10 +22,19 @@ def parse_args() -> argparse.Namespace:
         ),
     )
     p.add_argument(
-        '--start-db',
+        '--start-db-version',
         type=int,
         help='The starting DB version',
-        required=True,
+    )
+    p.add_argument(
+        '--start-db-hash',
+        type=str,
+        help='The starting DB commit hash',
+    )
+    p.add_argument(
+        '--start-db-path',
+        type=str,
+        help='The starting DB path',
```

Comment:
I find it confusing. What happens if I provide both db-path and db-hash? This should be documented in the help message and also avoid any such possible error. Also can't you make it so if you add the assets to also add the remote updates?